chaos_genius/utils/datetime_helper.py

```diff
@@ -50,20 +50,19 @@ def _get_tz_from_offset_str(utc_offset_str):
 
 def get_lastscan_string_with_tz(datetime_value_str) -> str:
     """Get last scan time in reporting timezone."""
-    reporting_tz_offset = SUPPORTED_TIMEZONES[TIMEZONE]
     server_tz_offset = timezone(datetime.now().astimezone().utcoffset())
 
     datetime_value = pd.Timestamp(
         datetime.strptime(datetime_value_str, "%Y-%m-%dT%H:%M:%S.%f")
     ).tz_localize(tz=server_tz_offset)
 
-    if reporting_tz_offset:
-        timezone_info = _get_tz_from_offset_str(reporting_tz_offset)
+    if TIMEZONE in SUPPORTED_TIMEZONES:
+        timezone_info = _get_tz_from_offset_str(SUPPORTED_TIMEZONES[TIMEZONE])
     else:
-        timezone_info = _get_tz_from_offset_str("GMT+00:00")
+        timezone_info = TIMEZONE
```

Comment:
I noticed that this code snippet is repeated in 2 places (here and in data loader). Would be better to make another util function for this.

Reply:
Yea good idea, but I think let's leave it like this for this PR.

I'll create a separate PR for it later.